File: app/src/main/java/com/miguel/armas/developer/lectorqrsinpublicidad/Entidades/HistorySqlLite.java
package com.miguel.armas.developer.lectorqrsinpublicidad.Entidades;

public class HistorySqlLite {


    private String id_history;
    private String save_string;
    private int date;
    public HistorySqlLite(){
    }
    public HistorySqlLite(String id_history, String save_string, int date) {
        this.id_history = id_history;
        this.save_string=save_string;
        this.date=date;
    }


    public String getid_history() {
        return id_history;
    }
    public void setid_history(String id_history) {
        this.id_history = id_history;
    }
    public String getsave_string() {
        return save_string;
    }
    public void setsave_string(String save_string) {
        this.save_string = save_string;
    }
    public int getdate() {
        return date;
    }
    public void setdate(int date) {
        this.date = date;
    }

}

File: app/src/main/java/com/miguel/armas/developer/lectorqrsinpublicidad/Entidades/HistorySqlLite.kt
package com.miguel.armas.developer.lectorqrsinpublicidad.Entidades

class HistorySqlLite {
    private var id_history: String? = null
    private var save_string: String? = null
    private var date = 0

    constructor() {}
    constructor(id_history: String?, save_string: String?, date: Int) {
        this.id_history = id_history
        this.save_string = save_string
        this.date = date
    }

    fun getid_history(): String? {
        return id_history
    }

    fun setid_history(id_history: String?) {
        this.id_history = id_history
    }

    fun getsave_string(): String? {
        return save_string
    }

    fun setsave_string(save_string: String?) {
        this.save_string = save_string
    }

    fun getdate(): Int {
        return date
    }

    fun setdate(date: Int) {
        this.date = date
    }
}